vayesta/core/qemb/qemb.py

```diff
@@ -1591,3 +1597,11 @@ def brueckner_scmf(self, *args, **kwargs):
         """Decorator for Brueckner-DMET."""
         self.with_scmf = Brueckner(self, *args, **kwargs)
         self.kernel = self.with_scmf.kernel.__get__(self)
+
+    def check_solver(self, solver):
```

Comment:
Duplicated from the fragment class?

Reply:
Just so we can check both whether the default configuration is valid at initialisation, and the specific configuration for each fragment. It could be amalgamated, but the checks are slightly different and it's a very short function